runtime/shared_common/OSCachesysv.cpp

```diff
@@ -190,7 +190,7 @@ SH_OSCachesysv::startup(J9JavaVM* vm, const char* ctrlDirName, UDATA cacheDirPer
 
 #if defined(WIN32)
 	_semFileName = _cacheNameWithVGen;
-#else
+#else /* !defined(WIN32) */
```

Comment:
The comment should match the expression in the `#if` directive, not its negation.
See https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-conditional-compilation-directives-if-ifdef-ifndef-else-elif-endif.